specs/challenge.md

```diff
@@ -229,9 +234,9 @@ Colosseum should be behind upgradable proxies.
 
 | Name                       | Value                                                              | Unit              |
 |----------------------------|--------------------------------------------------------------------|-------------------|
-| `MINIMUM_STAKE`            | TBD                                                                | gwei              |
-| `INTERACTION_TIMEOUT`      | TBD                                                                | seconds           |
-| `PROOF_TIMEOUT`            | TBD                                                                | seconds           |
+| `MINIMUM_BOND_AMOUNT`      | TBD                                                                | wei               |
+| `BISECTION_TIMEOUT`        | TBD                                                                | seconds           |
+| `PROVING_TIMEOUT`          | TBD                                                                | seconds           |
 | `PROOF_CORRECTION_TIMEOUT` | TBD                                                                | seconds           |
 | `SEGMENTS_LENGTHS`         | [9, 6, 10, 6]                                                      | array of integers |
```

Comment:
just trivial thing, it is declared as `segmentsLengths` in `Colosseum.sol`. also in [line 99](https://github.com/kroma-network/kroma/pull/53/files#diff-cbfa127e2342a83c72775dbe9b975ffab0548f12513492287f8cc22e380be3b2R99)
```suggestion
| `segmentsLengths`         | [9, 6, 10, 6]                                                      | array of integers |
```

Reply:
Since this is a specification, there is no need to worry about matching contract variable names.

Reply:
I see, thanks!